Ansible/ansible_install.sh

```
@@ -0,0 +1,3 @@
+sudo apt-add-repository ppa:ansible/ansible
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)